e2e/tests/components/select/default.spec.ts

```diff
@@ -0,0 +1,20 @@
+import { expect, test } from '../../../fixtures'
+
+test.describe('select, default', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('/components/select/default')
+  })
+
+  test('initial value is selected', async ({ page }) => {
+    await expect(page.getByLabel('Sort By')).toHaveValue('updated')
+  })
+
+  test('contains four options', async ({ page }) => {
+    await expect(page.getByRole('option')).toHaveCount(4)
+  })
+
+  test('selecting a new option updates the value', async ({ page }) => {
+    await page.getByLabel('Sort By').selectOption('comments')
+    await expect(page.getByLabel('Sort By')).toHaveValue('comments')
```

Comment:
Hmm I guess it's case insensitive, but probably would be good to keep the case the same anyway:
```suggestion
    await expect(page.getByLabel('Sort by')).toHaveValue('comments')
```